sdks/java/core/src/test/java/org/apache/beam/sdk/schemas/JavaFieldSchemaTest.java

```diff
@@ -728,4 +730,46 @@ public void testNoCreateOptionThrows() throws NoSuchSchemaException {
         thrown.getMessage(),
         containsString("zero-argument constructor"));
   }
+
+  @Test
+  public void testSelfNestedPOJOThrows() throws NoSuchSchemaException {
+    SchemaRegistry registry = SchemaRegistry.createDefault();
```

Comment:
nit: Add a space between creating the registry and getting the exception. Unit tests should follow the Arrange/Act/Assert design pattern (http://wiki.c2.com/?ArrangeActAssert), and we generally use spaces to separate the sections. You can see this in other tests in this file.